src/lib/bootloader/grub2.rb

```diff
@@ -149,9 +159,21 @@ def packages
         res << "syslinux"
       end
 
+      if Yast::Arch.x86_64
+        # FIXME: both? arch dependency?
+        res << "trustedgrub2" << "trustedgrub2-i386-pc" if @trusted_boot
+      end
+
       res
     end
 
+    # FIXME: refactor with injection like super(prewrite: prewrite, sysconfig = ...)
+    # overwrite BootloaderBase version to save trusted boot
+    def write_sysconfig(prewrite: false)
+      sysconfig = Bootloader::Sysconfig.new(bootloader: name, trusted_boot: @trusted_boot)
```

Comment:
this is not so nice as it write this sysconfig value even on non x86 architectures, so if someone check that file, he is confused